setup.py

```diff
@@ -157,6 +157,7 @@ class OpenLDAP2:
   install_requires=[
     'pyasn1 >= 0.3.7',
     'pyasn1_modules >= 0.1.5',
+    'typing_extensions',
```

Comment:
Should probably have a versioned dependency here. This is imported runtime and a too old version will cause `ImportError`.

Have a look in which version the symbols were added that you're using: https://github.com/python/typing_extensions/blob/main/CHANGELOG.md